src/org/infinity/gui/converter/ConvertToMos.java

```diff
@@ -933,34 +973,38 @@ private String pvrzInfoString(Object o)
     return String.format("Resulting in MOS%04d.PVRZ, MOS%04d.PVRZ, ...", index, index+1);
   }
 
-  private String getImageFileName(Path path)
+  private File[] getImageFileName()
   {
-    JFileChooser fc = new JFileChooser(path.toFile());
-    fc.setDialogTitle("Select input graphics file");
+    JFileChooser fc = new JFileChooser();
+
+    fc.setDialogTitle("Select input graphics files");
     fc.setDialogType(JFileChooser.OPEN_DIALOG);
     fc.setFileSelectionMode(JFileChooser.FILES_ONLY);
+    fc.setMultiSelectionEnabled(true);
     FileNameExtensionFilter[] filters = getInputFilters();
     for (final FileNameExtensionFilter filter: filters) {
       fc.addChoosableFileFilter(filter);
     }
     fc.setFileFilter(filters[0]);
     int ret = fc.showOpenDialog(this);
     if (ret == JFileChooser.APPROVE_OPTION) {
-      return fc.getSelectedFile().toString();
+      File[] filePaths = fc.getSelectedFiles();
+      for(File fileSelected : filePaths) {
+        System.out.println(fileSelected.getName());
+      }
```

Comment:
Use `Logger` or remove
```suggestion
      for (File fileSelected : filePaths) {
        System.out.println(fileSelected.getName());
      }
```